app/models/split.rb

```diff
@@ -143,6 +144,18 @@ def registry
     end
   end
 
+  def require_app_name_prefix=(value)
+    if instance_variable_defined?(:@require_app_name_prefix)
+      @require_app_name_prefix
```

Comment:
In case my other suggestion doesn't work, the setter doesn't need to return anything here - it will already return the passed value by default.

Reply:
oh right